eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,70 @@
 param (
-  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
-  $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  [string]$TargetDirectory, # Code path to code owners. e.g sdk/core/azure-amqp
+  [string]$RootDirectory = "$env:SYSTEM_DEFAULTWORKINGDIRECTORY", # The repo contains CODEOWNER file.
+  [string]$CodeOwnerFileLocation = "$PSSCriptRoot/../../../.github/CODEOWNERS", # The absolute path of CODEOWNERS file. 
+  [string]$ToolVersion = "1.0.0-dev.20211118.20", # Placeholder. Will update in next PR
+  [string]$ToolPath = "$env:AGENT_TOOLSDIRECTORY", # The place to check the tool existence. Put $(Agent.ToolsDirectory) as default
```

Comment:
Let's default this to a tmp directory. See https://github.com/Azure/azure-sdk-tools/pull/2245#discussion_r748583728

Reply:
I am thinking to use devop Tools directory by default. So we do not need to create folder in pipeline.
For local run, I have some workflow to create temp folder.

Reply:
I would default the value inline like we are doing in other places from that discussion https://github.com/Azure/azure-sdk-tools/pull/2245#discussion_r748583728